airbyte-integrations/connectors/source-shopify/source_shopify/schemas/abandoned_checkouts.json

```diff
@@ -173,10 +212,125 @@
     "line_items": {
       "items": {
         "properties": {
-          "applied_discounts": {},
-          "total_discount_set": {},
-          "pre_tax_price_set": {},
-          "price_set": {},
+          "applied_discounts": {
+            "type": ["null", "array"],
+            "items": {
+              "oneOf": [
+                {
+                  "type": ["null", "string"]
```

Comment:
here we should just make this a string

Reply:
Changed everywhere, where it was, to be a `string`.